store/postgres/inter_project_job_spec_repository.go

```diff
@@ -0,0 +1,49 @@
+package postgres
```

Comment:
Can you also add some unit tests for this file? The current one doesn't pass the pipeline.